packages/app/src/systems/Core/components/Layout/TopBar.tsx

```diff
@@ -0,0 +1,86 @@
+import { css } from "@fuel-ui/css";
+import {
+  FuelLogo,
+  Flex,
+  Icon,
+  IconButton,
+  Spinner,
+  Text,
+} from "@fuel-ui/react";
+import { useNavigate } from "react-router-dom";
+
+import { useLayoutContext } from "./Layout";
+
+type TopBarProps = {
+  title?: string;
+};
+
+export function TopBar({ title }: TopBarProps) {
+  const navigate = useNavigate();
+  const isInternal = Boolean(title);
+  const { isLoading } = useLayoutContext();
+
+  return (
+    <Flex as="nav" className={style({ isInternal })}>
+      <Flex css={{ alignItems: "center", gap: "$5", flex: 1 }}>
+        {isInternal ? (
+          <>
+            <IconButton
+              icon={<Icon icon="CaretLeft" color="gray8" />}
+              aria-label="Back"
+              variant="link"
+              css={{ px: "0 !important" }}
```

Comment:
Why do we need important here?

Reply:
Because that [shit error from Stitches](https://github.com/stitchesjs/stitches/issues/1060) I mentioned earlier about generated class precedence 😕